### ISSUE TITLE
[BUG/ISSUE]Cannot do the dry run on the AWS cloud in 13.2.1(no yaml module)

### ISSUE THREAD

**Issue by ruifeili** [geoschem/geos-chem#937]
### Description of the problem
I want to conduct the full chemistry simulation on AWS cloud. But when I follow the instruction, the problem occurred in the process of dry-run. It showed that" File "./download_data.py", line 40, in <module>  import yaml ModuleNotFoundError: No module named 'yaml'". 

### Description of troubleshooting performed
```
./download_data.py log.dryrun --aws
Traceback (most recent call last):
  File "./download_data.py", line 40, in <module>
    import yaml
ModuleNotFoundError: No module named 'yaml'
```

### GEOS-Chem version
13.2.1

### Description of modifications
Describe your modifications here.

### Log files
- Build log (if applicable):
- Run logs (if applicable):
- Run script (if applicable):

### Software versions
- CMake version:
- Compilers (Intel or GNU, and version):
- NetCDF version:

**Comment by laestrada** [comment]
Hi Ruifelli, 

The error looks like not all of the python packages needed are installed. Specifically the error you are hitting looks like PyYAML is not installed. Have you set up [gcpy](https://gcpy.readthedocs.io/en/stable/)? If so, do you have your conda environment activated?

**Comment by yantosca** [comment]
Thanks for writing @ruifeili.  I think @lizziel is correct -- you will need to activate a Conda environment before running the `download_data.py` script.

We would also encourage you to view [our Youtube video tutorial](https://i.ytimg.com/vi/ZLSXrjHqJAs/hqdefault.jpg?sqp=-oaymwEcCPYBEIoBSFXyq4qpAw4IARUAAIhCGAFwAcABBg==&rs=AOn4CLCre0CxeYmf-CPXwQ5QuFf8Tz4xow) that outlines the steps of how to do a dry-run in 13.2.1 and later versions.  That will walk you through the entire process.

**Comment by laestrada** [comment]
@ruifelli we have published the amis for 13.2.1 and 13.2.0. See [here](http://wiki.seas.harvard.edu/geos-chem/index.php/GEOS-Chem_versions) for the ami ids.

**Comment by ruifeili** [comment]
@yantosca @laestrada 
Thank you for the help. I used the AMI for 13.2.1 (ami-0983d1d6b06e4dd3f) to run the simulation. I watched the suggested video and typed the command "conda activate geo" and "time -p ./download_data.py log.dryrun amazon". Now it works. Thank you so much!
